kit/Kit.cpp

```diff
@@ -2090,13 +2089,12 @@ void Document::checkIdle()
     ProcessToIdleDeadline = std::chrono::steady_clock::now() - std::chrono::milliseconds(10);
 }
 
-void Document::enableProcessInput(bool enable)
+bool Document::processInputEnabled() const
 {
-    LOG_TRC("Document - input processing now: " <<
-            (enable ? "enabled" : "disabled") <<
-            " was " <<
-            (_inputProcessingEnabled ? "enabled" : "disabled"));
-    _inputProcessingEnabled = enable;
+    bool enabled = !_websocketHandler || _websocketHandler->processInputEnabled();
```

Comment:
I suppose it's intentional that when we have no socket we assume input processing is enabled.

Reply:
And yes; perhaps un-necessary, but I was bitten hard by this non processing of input ;-)